db/migrate/20160614185348_add_type_to_reservations.rb

```diff
@@ -1,9 +1,12 @@
 class AddTypeToReservations < ActiveRecord::Migration
 
+  class Reservation < ActiveRecord::Base
```

Comment:
Can we add a comment here and below that this class is being added to avoid issues with the default scope from paranoia?